homeassistant/components/emoncms/sensor.py

```diff
@@ -252,48 +221,25 @@ def update(self) -> None:
             return
 
         self._elem = elem
-
-        if self._value_template is not None:
-            self._state = self._value_template.render_with_possible_json_value(
-                elem["value"], STATE_UNKNOWN
-            )
-        elif elem["value"] is not None:
-            self._state = round(float(elem["value"]), DECIMALS)
-        else:
-            self._state = None
+        self._state = elem["value"]
 
 
 class EmonCmsData:
     """The class for handling the data retrieval."""
 
-    def __init__(self, hass, url, apikey, interval):
+    def __init__(self, url: str, apikey: str) -> None:
         """Initialize the data object."""
-        self._apikey = apikey
+        self._sess = requests.Session()
```

Comment:
This would not be allowed. See https://developers.home-assistant.io/docs/development_checklist